pandas/core/groupby.py

```diff
@@ -1247,6 +1247,22 @@ def f(self, **kwargs):
                         result = result._convert(datetime=True)
                     return result
 
+            # Two cases to get the signature correct for sum / prod
+            if min_count == -1:
+                @Substitution(name='groupby', f=name)
+                @Appender(_doc_template)
+                @Appender(_local_template)
+                def f(self, **kwargs):
+                    return f_(self, **kwargs)
+            else:
```

Comment:
? not sure u need this
shouldn’t the template just work for this?

Reply:
It's not required, but with it I get the `min_count` in the signature.

```python
In [5]: gr.sum?
Signature: gr.sum(min_count=1, **kwargs)
Docstring:
Compute sum of group values
```

Without the if / else, things work , but then the sig is just `**kwargs`. I have a slight preference for including it the signature, but will revert it if you want.

Reply:
I would revert it for now. we don't include this for anything else (IOW any parameters now).

Reply:
IIRC there is an issue about this. its needs a comprehensive soln.

Reply:
Yeah, I forgot about numeric_only, etc. OK.